front/types/chat.ts

```diff
@@ -36,7 +36,7 @@ export type ChatSessionType = {
   created: number;
   sId: string;
   title: string | null;
-  messages?: ChatMessageType[];
+  messages: ChatMessageType[] | null;
```

Comment:
I would go for `[]` as null value so `messages: ChatMessageType[]` ?

Reply:
But I guess our current implemetnation rely on the possibility of null.

Reply:
yes it does, it's risky because in JS `[]` isn't falsy